localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/resource_router.py

```diff
@@ -94,6 +96,16 @@ def route_and_enrich(self, context: RestApiInvocationContext):
         )
         context.resource_method = method
 
+        self.update_context_variables_with_resource(context.context_variables, resource)
+
+    @staticmethod
+    def update_context_variables_with_resource(
+        context_variables: ContextVariables, resource: Resource
+    ):
+        # TODO: log updating the context_variables?
```

Comment:
It is fine to leave as a TODO, but I think it would really useful to log them in debug mode.

Reply:
Done 👍